bld/namelist_files/namelist_definition_mpassi.xml

```diff
@@ -3715,3 +3728,4 @@ Default: Defined in namelist_defaults.xml
 </entry>
 
 </namelist_definition>
+dazlich@derecho4:~/ewv2.1/mpassi-pres/components/mpas-seaice> 
```

Comment:
Need to remove this

Reply:
oops, fixed

Reply:
Yep, fixed by [eb5cf9f](https://github.com/EarthWorksOrg/mpas-seaice/pull/8/commits/eb5cf9f1fa424cb809bc985cb197353f0ef204df)